sandbox/settings.py

```diff
@@ -335,6 +344,9 @@ class Development(Base):
                 "propagate": True,
             },
             "django": {"handlers": ["console"], "level": "INFO", "propagate": True},
+            # This formatter is here as an example to what is possible to do
+            # with xapi loggers.
+            "xapi": {"handlers": ["gelf"], "level": "INFO", "propagate": True},
```

Comment:
I've set the `INFO` level because the xapi logger is currently used with the info level only, and this is just an example.
I can change it if you think it's better with `DEBUG`, I confess have no particular opinion on this point ^^

Reply:
Alright, so let's keep `INFO` level :wink: